open-api.yaml

```diff
@@ -519,35 +519,75 @@ components:
       schema:
         type: number
         default: 0
+    q:
+      name: q
+      in: query
+      description: Query string.
+      schema:
+        type: string
+        example: back to the future
+        default: '""'
     attributesToRetrieve:
       name: attributesToRetrieve
       in: query
-      description: |
-        Comma-separated list of attributes whose fields will be present in the returned documents.
-        Defaults to to the [displayedAttributes list](/reference/features/settings.html#displayed-attributes) which contains by default all attributes found in the documents.
+      description: 'Comma-separated list of attributes whose fields will be present in the returned documents. Defaults to to the [displayedAttributes list](/reference/features/settings.html#displayed-attributes) which contains by default all attributes found in the documents.'
```

Comment:
This definition is only in the context of a `GET /search` ? I'm not I saw the method attached to it.


Same for all array parameters

Reply:
Yes! You can test it over `get`and `post` `/search` routes. GET does not handle array syntax and POST does not handle string syntax separated by `,`

Reply:
in the yaml it is not clear that this is for the `GET` method. Unless I didn't see it

Reply:
Query parameters are declared as components so they can be reused. They are linked using #ref to the routes using it!